src/main/java/me/guichaguri/tickratechanger/TickrateContainer.java

```diff
@@ -46,6 +46,7 @@
 
     @EventHandler
     public void preInit(FMLPreInitializationEvent event) {
+		new TickrateChanger();

```

Comment:
FML should initialize it to apply the transformer, for dev environment you need `-Dfml.coreMods.load=me.guichaguri.tickratechanger.TickrateChanger` in the JVM arguments.
If this this class is not initialized by FML, the mod doesn't work.